tests/e2e/fixtures/test-fixtures.ts

```
@@ -0,0 +1,16 @@
+import { test as base } from '@playwright/test';
+import { ElectronAppHelper } from '../helpers/electron-app';
+
+/**
+ * Custom test fixture that provides automatic Electron app lifecycle management
+ */
+export const test = base.extend<{ appHelper: ElectronAppHelper }>({
+  appHelper: async ({}, use) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)